neo4j/_async/work/session.py

```diff
@@ -40,7 +43,7 @@
     TransactionError,
 )
 from ...work import Query
-from .result import AsyncResult
+from .result import AsyncResult, QueryResult
```

Comment:
Please install the pre-commit hook as explained [here](https://github.com/neo4j/neo4j-python-driver/blob/5.0/CONTRIBUTING.md#specifically-for-this-project).

It would've auto-formatted this import properly for you :)

```python
from .result import (
    AsyncResult,
    QueryResult,
)
```

Reply:
It is installed, but the changes was mad previous the installation.